### ISSUE TITLE
Zmiana z WzA

### ISSUE THREAD

**Issue by jesiiu** [allegro/allegro-api#9048]
Witam, 
Mam pytanie odnośnie zmian w API WzA związanych z InPost. Czy od 23 maja będą wymagane zmiany w API?
Przykładowo:
2488f7b7-5d1c-4d65-b85c-4cbcf253fd93#id_umowy_wlasnej dla metod Allegro Paczkomaty InPost - oznacza to, że delivery_method_id będzie wyglądać  w ten sposób w '/shipment-management/delivery-services' i nie trzeba będzie przekazywać credentials_id do metody generującej etykiety czy trzeba będzie to połączyć tym # przed wysłaniem zapytania?

**Comment by MartaNowaczyk** [comment]
W `/shipment-management/delivery-services` zostanie po staremu czyli będzie oddzielnie "deliveryMethodId" i umowa własna w "credentialsId". W zapytaniu musi iść natomiast w odpowiedniej strukturze z #.

**Comment by jesiiu** [comment]
Czyli jeżeli dotychczas zapytanie wyglądało tak
                            'input' => [
                                'deliveryMethodId' => $deliveryData['delivery_method_id'],
                                'credentialsId' => $deliveryData['credentials_id'] ?? null,
                                
To od 23 maja dla inpost powinno wyglądać 
                            'input' => [
                                'deliveryMethodId' => $deliveryData['delivery_method_id']. '#' . $deliveryData['credentials_id'] ?? null,
                                'credentialsId' => $deliveryData['credentials_id'] ?? null,
                                
                                Czy w takim razie pole credentialsId jest potrzebne osobno czy można je wyciąć?
                                Po przesłaniu w formie z # rozliczenia będą po stawkach allegro?

**Comment by jesiiu** [comment]
Czyli jeżeli moje zapytanie wygląda w ten sposób
'input' => [
'deliveryMethodId' => $deliveryData['delivery_method_id'],
'credentialsId' => $deliveryData['credentials_id'] ?? null,

To nic nie muszę zmieniać i będzie prawidłowo rozliczone po 23 maja, tak?

**Comment by AureliuszBrussy** [comment]
Tak, będzie to prawidłowe - nie musisz nic zmieniać.

**Comment by jesiiu** [comment]
Oczywiście prawidłowo rozliczone to mam na myśli, że rozliczone po stawkach allegro, tak?

**Comment by MartaNowaczyk** [comment]
Tak

**Comment by jesiiu** [comment]
{
      "delivery_method_id":"321...",
      "credentials_id":"123..",
      "delivery_name":"Kurier InPost (inpost1)"
   },
   {
      "delivery_method_id":"321..",
      "credentials_id":"123..",
      "delivery_name":"Kurier InPost pobranie (inpost1)"
   },
   {
      "delivery_method_id":"321..",
      "credentials_id":"123..",
      "delivery_name":"Paczkomaty InPost (inpost1)"
   },
   {
      "delivery_method_id":"321..",
      "credentials_id":"123..",
      "delivery_name":"Paczkomaty InPost pobranie (inpost1)"
   },
   
   Wyświetliłem sobie dane z zasobu "/shipment-management/delivery-services" i wygląda to mniej więcej w ten sposób, dokładnie tak samo jak przed "zmianami" z InPost, prosiłbym tylko o potwierdzenie, że wszystko jest poprawnie i przesyłki będą rozliczane przez stawki allegro.
   ID są zmienione przeze mnie, nigdzie nie było znaku '#' w id

**Comment by MartaNowaczyk** [comment]
tak, nie będzie tam # w ID. Zmiany o których pisaliśmy dotyczą metod z Allegro na początku: 
- Allegro Paczkomaty InPost 
- Allegro Paczkomaty InPost pobranie
- Allegro Kurier24 InPost 
- itd.
ogólnie sprawa wygląda tak, że przekazujesz metodę dostawy, którą masz w zamówieniu (no bo musi być zgodna z zamówieniem), do tego dodajesz odpowiednią umowę i wtedy będzie to odpowiednio rozliczone. Czyli praktycznie jak przed zmianami.
